README.md

```diff
@@ -203,6 +211,34 @@ To generate from the command line on the Foreman server:
 Set `OUTPUT=/path` to change the output destination path (directory or file).
 It must be writable by the 'foreman' user.
 
+### Subnet images
+
+Subnet image is similar to generic image, but chain-loading is done via TFTP
+Smart Proxy assigned to Subnet of the host. The smart proxy must have
+"Templates" plugin installed, enabled and configured to report http URL (https
+is not supported by iPXE at the moment).
+
+This image is generic for all hosts from the given subnet.
+
+To generate the image from the web interface, view a host page, click the
+"Boot disk" button and select "Subnet image".
+
+To generate using the Hammer CLI, install the [hammer_cli_foreman_bootdisk](https://github.com/theforeman/hammer_cli_foreman_bootdisk)
+plugin and run:
+
+    hammer bootdisk subnet
```

Comment:
https://github.com/theforeman/hammer_cli_foreman_bootdisk/pull/4 suggests it's a flag under host?